bucket-handlers.go

```diff
@@ -540,7 +540,7 @@ func (api objectAPIHandlers) HeadBucketHandler(w http.ResponseWriter, r *http.Re
 	}
 
 	if _, err := api.ObjectAPI.GetBucketInfo(bucket); err != nil {
-		errorIf(err, "GetBucketInfo failed.", nil)
+		errorIf(err, "Unable to fetch bucket info.")
 		writeErrorResponse(w, r, toAPIErrorCode(err), r.URL.Path)
```

Comment:
In consistent "." ending. If errorIf doesnt automatically insert it, we should always explicitly add it.